src/ansys/fluent/core/workflow.py

```diff
@@ -328,12 +328,18 @@ def python_name(self) -> str:
             The Pythonic name of this task.
```

Comment:
```suggestion
            Pythonic name of the task.
```
Can " , as it is in the underlying application" be changed to "from the underlying application" instead? For example, "Get the Pythonic name of this task from the underlying application."